pkg/omf/functions/compat/refresh.fish

```diff
@@ -0,0 +1,14 @@
+function refresh -d "(deprecated) Refresh fish session by replacing current process"
+  printf 'warning: function %srefresh%s is deprecated and will be removed soon.\n %s\n' \
+  (set_color -u) (set_color normal) \
+  (status -t)[5] >&2
```

Comment:
I could split the stack trace and the index expansion to make it clearer what `status -t` output is, or I could use `status --print-stack-trace`.